file/writable_file_writer.cc

```diff
@@ -187,7 +187,20 @@ IOStatus WritableFileWriter::Flush() {
     }
   }
 
-  s = writable_file_->Flush(IOOptions(), nullptr);
+  {
+    IOSTATS_TIMER_GUARD(write_nanos);
+    TEST_SYNC_POINT("WritableFileWriter::Flush:BeforeAppend");
+    FileOperationInfo::TimePoint start_ts;
+    if (ShouldNotifyListeners()) {
+      start_ts = std::chrono::system_clock::now();
+    }
+    s = writable_file_->Flush(IOOptions(), nullptr);
+    if (ShouldNotifyListeners()) {
+      auto finish_ts = std::chrono::system_clock::now();
+      NotifyOnFileFlushFinish(next_write_offset_, buf_.CurrentSize(), start_ts,
```

Comment:
`buf_.CurrentSize()` may not be the size of the IO. For `Flush()`, we do not have to report it now if it's hard to get the actual size.